content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx

```diff
@@ -68,7 +68,7 @@ Either side of the relation can be backed by a foreign key. In the following exa
 model User {
```

Comment:
Unrelated: When checking this change, I noticed how werid the wording in line 62 is. This example here is the same we used in lines 22-28, but that is not acknowledged or mentioend so I kept checking if I was missing something. Maybe we can add something along the lines of "(this is the same example that was used further up the page)" or similar.

Reply:
Agree this is a bit weird. Hopefully worded better now.